src/labkey/query/Rows.ts

```diff
@@ -232,6 +232,17 @@ export interface Command {
     schemaName: string;
 }
 
+export interface  MoveRowsResponse extends ModifyRowsResults {
+    /** The container path in which the rows were moved. */
+    containerPath: string;
```

Comment:
Should `containerPath` really be marked as required?

Reply:
Changed to optional.